src/test/java/com/jcabi/aspects/aj/MnemosTest.java

```diff
@@ -52,7 +57,7 @@ public void buildsTextFromObject() throws Exception {
             new Object[] {"\u20ac-plain", "'\u20ac-plain'"},
             new Object[] {"test ", "'test '"},
             new Object[] {null, "NULL"},
-            new Object[] {new String[0], "[]"},
+            new Object[] {new String[0], EMPTY_ARRAY},
```

Comment:
Every reference to a `static` field should have class in front, so here it would be `MnemosTest.EMPTY_ARRAY`